protocol/chat.go

```diff
@@ -150,7 +150,16 @@ func (c *Chat) MembersAsPublicKeys() ([]*ecdsa.PublicKey, error) {
 	return stringSliceToPublicKeys(publicKeys, true)
 }
 
-func (c *Chat) updateChatFromProtocolGroup(g *v1protocol.Group) {
+func (c *Chat) updateChatFromProtocolGroup(myID string, g *v1protocol.Group) {
```

Comment:
The only real issue I've got is the name of the function. It should probably change as it is no longer `updateChatFromProtocolGroup(g *v1protocol.Group)` its now something like `updateChanges(myID string, g *v1protocol.Group)`.

What do you think?